Sources/Storage/Disk/Extension/String+FileURL.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  String+FileURL.swift
+//  StatsdClient
+//
+//  Created by Nghia Tran on 10/9/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+
+    func fileURL() -> URL {
```

Comment:
I don't see the reason for having this extension. It wraps 1 line of code which isn't even that hard.
Also I think `URL(fileURLWithPath: "file://somefolder")` is even easier to understand than `"file://somefolder".fileURL()` since you know it will return a `URL` type